ocfweb/docs/docs/services/web/jekyll.md

```diff
@@ -0,0 +1,63 @@
+[[!meta title="Jekyll"]]
+
+[Jekyll](https://jekyllrb.com) is a popular static website generator.
+
+## Set up
+
+1. Go to our [web-based SSH client](https://ssh.ocf.berkeley.edu/) and sign in
+   with your username and password. Or, at terminal, SSH into tsuanami.ocf.berkeley.edu.
+
+2. Install Jekyll locally:
+```shell
+  gem install jekyll --user-install
+```
+
+3. Source the executable:
+```shell
+  echo "PATH=$PATH:~/.gem/ruby/2.1.0/bin">>~/.bash_profile
```

Comment:
any way we can avoid hard-coding the version? this will presumably break when we upgrade, and it's unlikely we'll notice until somebody complains (could be months or years)